spyder/app/mainwindow.py

```diff
@@ -341,17 +342,17 @@ def register_plugin(self, plugin):
             margin = 0
             if CONF.get('main', 'use_custom_margin'):
                 margin = CONF.get('main', 'custom_margin')
-
             plugin.update_margins(margin)
 
         # First time plugin starts
         if plugin.get_conf_option('first_time', True):
             logger.info("Tabify {} dockwidget...".format(plugin.NAME))
-            if isinstance(plugin, SpyderDockablePlugin):
+            if (isinstance(plugin, SpyderDockablePlugin)
+                    and plugin.NAME != Plugins.Console):

```

Comment:
This fix is not good enough for the internal console itself, if it was visible in the previous session and you open Spyder on this branch. Look what happens:

![Selección_053](https://user-images.githubusercontent.com/365293/83040077-7dda6700-a004-11ea-8c1a-e3606dca3ff9.png)

Please fix that.

Reply:
Will fix it!